### ISSUE TITLE
can not use it with typescript

### ISSUE THREAD

**Issue by fanyuzzz** [konvajs/use-image#8]
```
const [image] = useImage('...');
return <Image image={image} />
```
 the typescript warning:
Type 'HTMLImageElement | undefined' is not assignable to type 'ImageBitmapSource'.
  Type 'undefined' is not assignable to type 'ImageBitmapSource'.ts(2322)
Image.d.ts(4, 5): The expected type comes from property 'image' which is declared here on type 'IntrinsicAttributes & ImageConfig & KonvaNodeEvents & ClassAttributes<Image> & { children?: ReactNode; }'

**Comment by fanyuzzz** [comment]
```
const [image] = useImage('...');
const imageNew = image || document.createElement('img');
return <Image image={imageNew} />
```
it works with typescript like this!

**Comment by lavrton** [comment]
Never do this:
```
const imageNew = image || document.createElement('img');
```

Image may be rendered MANY times. Each time it will create `<img />` DOM node.

Any way. The fix on the `konva` side was required. I just published `konva@4.0.14`. It should work ok now. Can you try it?
